index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>h</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] `required`
- [ ] good job. identifying the title error :clap:, however, it's important to tell the student `why` you asked them to make changes because that's when they learn when you explain to them and that's how you help the student become a better developer.
- [ ] Also I suggest you ask the student to make the change more politely. using words like `please`, `kindly`, and `I suggest` and other words can be polite because we don't want to sound rude and discourage the student.
- [ ] Also the use of `emoji's` comes in handy please take advantage of because it helps you set a tone and sound more polite.